src/main/java/com/laserfiche/api/client/model/GetAccessTokenResponse.java

```diff
@@ -1,17 +1,120 @@
 package com.laserfiche.api.client.model;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
```

Comment:
Now I see why have some new dependencies. I am sorry but the client core library isn't auto generated. Can we just hand-write this class (you can just remove the `@javax.annotation.Generated` annotation and stuff like `@Schema(description = "")` and the unnecessary `equals()`, `toString()`, `hashCode()`.